### PR TITLE
feature/PS-106-discord-webhook-saving-frontend

### PR DESCRIPTION
**Pull request by AyserJamshidi**

### CODE REVIEW

**Comment by AyserJamshidi** [comment]
This branch related to the Creator settings page functionality.

File: gateway/src/main/java/com/patreonshout/beans/PostBean.java [hunk 1]
```diff
@@ -37,53 +37,63 @@ public class PostBean extends BaseResource {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name="post_id")
-    protected int post_id;
+    protected int postId;
+
+    @Column(name="app_id")
```
Comment (apsawicki) [inline gateway/src/main/java/com/patreonshout/beans/PostBean.java:42]: add javadocs for these

File: gateway/src/main/java/com/patreonshout/rest/WebhookSvc.java [hunk 2]
```diff
@@ -139,6 +125,24 @@ public String WebhookReceiver(
 		// Webhook
 		return "";
 	}
+
```
Comment (apsawicki) [inline gateway/src/main/java/com/patreonshout/rest/WebhookSvc.java:128]: add javadocs

File: gateway/src/test/java/rest/WebhookSvcTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ * Testing the WebhookSvc Endpoint
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration
+@DataJpaTest
+public class WebhookSvcTest {
+
+    @Autowired
```
Comment (apsawicki) [inline gateway/src/test/java/rest/WebhookSvcTest.java:26]: add javadocs

File: gateway/src/test/java/rest/WebhookSvcTest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    PostsRepository postsRepository;
+
+    @Test
```
Comment (apsawicki) [inline gateway/src/test/java/rest/WebhookSvcTest.java:29]: add javadocs

**Review by apsawicki — CHANGES_REQUESTED** [review]

**Review by AyserJamshidi — COMMENTED** [review]
Changes made.

**Review by apsawicki — APPROVED** [review]

**Pull request merged**